### PR TITLE
createdb: add page

### PR DESCRIPTION
**Pull request by counterposition**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented:** 14.2

### CODE REVIEW

File: pages/common/createdb.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# createdb
+
+> Create a PostgreSQL database.
+> More information: <https://www.postgresql.org/docs/current/app-createdb.html>.
+
+- Create a database owned by the current user:
+
+`createdb {{database}}`
```
Comment (navarroaxel) [inline pages/common/createdb.md:8]:
```suggestion
`createdb {{database_name}}`
```

File: pages/common/createdb.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Create a database owned by a specific user, with an optional description:
+
+`createdb -O {{user}} {{database}} 'Description of the database'`
```
Comment (navarroaxel) [inline pages/common/createdb.md:12]: We prefer long arguments if available
```suggestion
`createdb --owner={{username}} {{database_name}} '{{description}}'`
```

File: pages/common/createdb.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Create a database from a template:
+
+`createdb -T {{template_database}} {{database}}`
```
Comment (navarroaxel) [inline pages/common/createdb.md:16]:
```suggestion
`createdb --template={{template_name}} {{database_name}}`
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/createdb.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# createdb
+
+> Create a PostgreSQL database.
+> More information: <https://www.postgresql.org/docs/current/app-createdb.html>.
+
+- Create a database owned by the current user:
+
+`createdb {{database_name}}`
+
+- Create a database owned by a specific user, with an optional description:
```
Comment (CleanMachine1) [inline pages/common/createdb.md:10]:
```suggestion
- Create a database owned by a specific user with a description:
```

The command is based around having a description. Plus after removing the word optional, it reads better without the comma.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**